pkg/cmd/server/api/v1/conversions.go

```diff
@@ -29,6 +29,7 @@ func addDefaultingFuncs(scheme *runtime.Scheme) {
 				obj.PolicyConfig.OpenShiftInfrastructureNamespace = bootstrappolicy.DefaultOpenShiftInfraNamespace
 			}
 			if len(obj.RoutingConfig.Subdomain) == 0 {
+				// TODO: use dnsDomain instead of cluster.local
```

Comment:
This shouldn't be a TODO - the value has a standard default that is always applied.  We can't change a default once we declare it.